scout/server/blueprints/phenotypes/controllers.py

```diff
@@ -1,20 +1,15 @@
 # -*- coding: utf-8 -*-
 
-def hpo_terms(store, query = None, limit = None):
+def hpo_terms(store):
```

Comment:
That's one long page, but it seems to work just fine! I don't think ppl will be reloading this page too often either.